index.d.ts

```diff
@@ -43,6 +43,15 @@ export interface CellAttributes {
   getFromDOM?: getFromDOM;
   setDOMAttr?: setDOMAttr;
 }
+export enum NodeNames {
```

Comment:
lint? 2 spaces, space after and before =, and no trailing comma on label? we could add linting to entire repo